docs/installing-northstar/using-northstar/launch-arguments.md

```diff
@@ -11,5 +11,9 @@ Here's a list of new command line arguments that Northstar introduces, they shou
 | `-waitfordebugger` | Waits for debugger to connect before launching game                                                |                                  |
 | `-language`        | Forces loading of client localisation                                                              | language string ex: `portuguese` |
 | `-profile=`        | Enabled specifying a different profile directory. Default: R2Northstar                             | Example: `-profile="dev mods"`   |
-| \`LFX              | Enables the use of [LatencyFleX](../../using-northstar/playing-on-linux/#latencyflex) (Linux-only) | `0` or `1`                       |
-| `-enablechathooks` | Enables the use of chathooks for use by mods                                                       |                                  |
\ No newline at end of file
+| `-enablechathooks` | Enables the use of chathooks for use by mods                                                       |                                  |
+
+# Environment Variables
+| Variable name      | Description                                                                                        | Value                            |
+| ------------------ | -------------------------------------------------------------------------------------------------- | -------------------------------- |
+| `LFX`              | Enables the use of [LatencyFleX](../../using-northstar/playing-on-linux/#latencyflex) (Linux-only) | `0` or `1`                       |
```

Comment:
This link is broken

Reply:
Oh no. Considering that @emma-miler didn't change the path I guess the path was already broken before :I

Reply:
Checking blame it seems to have been me when doing it via gitbook webapp which is hilarious tbh ^^

https://github.com/R2Northstar/NorthstarWiki/blame/d5cfdb1dc644748f247da06c745cc898e1b3e4f4/docs/installing-northstar/using-northstar/launch-arguments.md